File: TiaCodegen/Commands/Coils/Coil.cs
using TiaCodegen.Commands.Signals;
using TiaCodegen.Interfaces;

namespace TiaCodegen.Commands.Coils
{
    public class Coil : BaseCoil
    {
        public Coil(Signal signal, IOperationOrSignal op = null) : base(signal, op)
        { }
    }
}

File: TiaCodegen-ts/src/Commands/Coils/Coil.ts
import { IOperationOrSignal } from '../../Interfaces/IOperationOrSignal';
import { Signal } from '../Signals/Signal';
import { BaseCoil } from './BaseCoil';

export class Coil extends BaseCoil {
    constructor(signal: Signal, op: IOperationOrSignal | null = null) {
        super(signal, op);
    }
}
